index.js

```diff
@@ -53,7 +55,15 @@ function runner (opts) {
       }
 
       if (req.url == '/') {
-        fs.createReadStream(__dirname + '/static/index.html').pipe(res);
+        if (opts.injectvdom) {
+          var vdom = injectVdom(opts.injectvdom);
+          vdom.pipe(res);
+        }
+        if (opts.injectcss) {
+          var css = injectCss(opts.injectcss);
+          css.pipe(opts.injectvdom ? vdom : res);
+        }
+        fs.createReadStream(__dirname + '/static/index.html').pipe(opts.injectcss ? css : opts.injectvdom ? vdom : res);
```

Comment:
this can be refactored to have less ternary operators:

```js
var stream = fs.createReadStream(__dirname + '/static/index.html');
if (opts.injectvdm) stream = stream.pipe(injectVdom(opts.injectvdom));
if (opts.injectcss) stream = stream.pipe(injectCss(opts.injectcss));
stream.pipe(res);
```

`Stream#pipe` will return the stream that was piped to.

Reply:
Yup, I think that's great. I wrote first in such a line but decide to change it. But I guess that the whole reason of ```trumpet``` design.